.github/workflows/_deploy_kafka_test.yml

```diff
@@ -0,0 +1,198 @@
+name: Setup Workload Identity 
+
+on:
+  workflow_call:
+    inputs:
+      skr-client-akv-endpoint:
+        description: "AKV endpoint for key release"
+        required: true
+        type: string
+      skr-client-maa-endpoint:
+        description: "Attestation endpoint"
+        required: true
+        default: "sharedeus2.eus2.test.attest.azure.net"
+        type: string
+      skr-client-kid:
+        description: "Key release key id"
```

Comment:
I would just have this hard-coded or auto-generated. Use "kafka-demo-key-randomnumber"

Reply:
I think we can recycle the key because they can be re-used. I will have the key hard coded.